### PR TITLE
census events jupyter docs

### PR DESCRIPTION
**Pull request by cdquinto**
Updated census events docs to match new conventions. This PR will close https://github.com/tskit-dev/msprime/issues/1271

### CODE REVIEW

**Comment by AdminBot-tskit** [comment]
📖 Docs for this PR can be previewed [here](https://tskit-dev.github.io/msprime-docs/PR-1306/)

File: docs/ancestry.rst [hunk 1]
```diff
@@ -1045,7 +1043,7 @@ event. At generation 100 and earlier, the two populations exchange migrants at a
 The resulting tree sequence has nodes on each tree at the specified census time.
 These are the nodes with IDs 8, 9, 10, 11, 12 and 13:
 
-.. code-block:: python
+.. jupyter-execute::
```
Comment (jeromekelleher) [inline docs/ancestry.rst:1046]: We're displaying the same tree sequence twice now, so we can delete the image block below. Can you also delete the file ``_static/ts_with_census_nodes.svg`` please?

Comment (jeromekelleher) [reply docs/ancestry.rst:1046]: Elsewhere in the docs were just using 
```
SVG(ts.draw_svg())
```
without the imports. I think we can probably just put the ``SVG(ts.draw_svg())`` at the end of the previous jupyter block too, there's no need for a separate box now.

**Review by jeromekelleher — COMMENTED** [review]
Looks great, thanks @cdquinto! Some minor comments.

**Comment by jeromekelleher** [comment]
It looks like the email address you're using in your local git config isn't the same as the one you use for GitHub @cdquinto. This would mean that your contributions wouldn't be correctly attributed to you [here](https://github.com/tskit-dev/msprime/graphs/contributors). The GitHub [docs](https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-your-github-user-account/setting-your-commit-email-address) for this are helpful.

**Comment by cdquinto** [comment]
Ok, thanks for the comments. I think I fixed everything, including my email.

**Review by jeromekelleher — APPROVED** [review]
Looks great, thanks @cdquinto! 

Welcome to the msprime contributors club!

**Pull request merged**